### PR TITLE
docs(stacked-notification): add knobs to storybook

### LINKED ISSUE #115: Cannot modify props of StackedNotification with knobs in Storybook
#### Screenshot

image

As seen in screenshot, the knobs are not there to modify the content inside StackedNotification.

### PR DESCRIPTION
**Pull request by kleyu**
## Affected issues

- Resolves #115

## Proposed changes

- Modified props to be in Storybook fashion with knobs available.
- Added `centered` decorator

## Testing instructions

- visit storybook

## Screenshot
image

### CODE REVIEW

File: src/components/StackedNotification/StackedNotification.js [hunk 1]
```diff
@@ -45,7 +46,7 @@ StackedNotification.propTypes = {
   title: PropTypes.string.isRequired,
   subtitle: PropTypes.node.isRequired,
   role: PropTypes.string,
-  caption: PropTypes.node,
+  caption: PropTypes.oneOfType([PropTypes.node, PropTypes.string]),
```
Comment (jendowns) [inline src/components/StackedNotification/StackedNotification.js]: A `node` type includes a `string` so no need to change this.

```suggestion
caption: PropTypes.node,
```

Comment (kleyu) [reply src/components/StackedNotification/StackedNotification.js]: Wasn't aware of that @jendowns, thanks! I just changed it back.

**Pull request merged**